react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -94,33 +103,35 @@ public void responseHtml(String title, String html, boolean changed) {
             @Override
             public void requestMediaPickFromMediaLibrary(MediaSelectedCallback mediaSelectedCallback) {
                 mPendingMediaSelectedCallback = mediaSelectedCallback;
-                onMediaLibraryButtonListener.onMediaLibraryButtonClicked();
+                mOnMediaLibraryButtonListener.onMediaLibraryButtonClicked();
             }
 
             @Override
             public void requestMediaPickFromDeviceLibrary(MediaUploadCallback mediaUploadCallback) {
                 mPendingMediaUploadCallback = mediaUploadCallback;
-                onMediaLibraryButtonListener.onUploadMediaButtonClicked();
+                mOnMediaLibraryButtonListener.onUploadMediaButtonClicked();
             }
 
             @Override
             public void requestMediaPickerFromDeviceCamera(MediaUploadCallback mediaUploadCallback) {
                 mPendingMediaUploadCallback = mediaUploadCallback;
-                onMediaLibraryButtonListener.onCapturePhotoButtonClicked();
+                mOnMediaLibraryButtonListener.onCapturePhotoButtonClicked();
             }
 
             @Override
             public void mediaUploadSync(MediaUploadCallback mediaUploadCallback) {
                 mPendingMediaUploadCallback = mediaUploadCallback;
-                onReattachQueryListener.onQueryCurrentProgressForUploadingMedia();
+                mOnReattachQueryListener.onQueryCurrentProgressForUploadingMedia();
```

Comment:
Wondering if there could be a case where this listener can be called before the view is attached to a container. In that case we need to check that are not null(s)?